shared/src/common/enums/api/bs-api-path.enum.ts

```diff
@@ -1,6 +1,7 @@
 enum BSApiPath {
   ROOT = '/',
   SPACES = '/spaces',
+  SPACES_FILE = '/spaces/:id/objects',
```

Comment:
```suggestion
  SPACES = '/spaces',
```
create a separate enum as for `spaces`